### PR TITLE
added graph2

### PR DESCRIPTION
**Pull request by mboom**

### CODE REVIEW

**Comment by DaanvanderValk** [comment]
The good news: this is a clear pull request, everything works and runs fine.
The bad news: the graph is empty; there should be content to confirm it's working.

File: src/modules/graph2.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            .domain([d3.max(stacked[stacked.length - 1], function (d) { return d.y0 + d.y; }), 0])
+            .range([0, h - padBottom - padTop]);
+    }
+
+    return {
+        name: 'graph2',
+        title: 'Session durations per pull-request',
+        xAxisLabel: 'Pull-requests',
+        yAxisLabel: 'Sessions and session duration',
+        parentSelector: '#project-modules',
```
Comment (agudek) [inline src/modules/graph2.js]: This probably fits better in the personal tab

**Comment by agudek** [comment]
Added some data to the server to see how it works and it doesn't seem to work correctly

**Comment by borek2** [comment]
On the graph part or on the data apart?

**Comment by agudek** [comment]
@borek2 a rect for the full session is drawn, but is not divided into semantic sessions. Could be caused by both parts

**Comment by mboom** [comment]
I will take a look at that.

**Pull request merged**